pkgs/development/python-modules/instaloader/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, sphinx
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "instaloader";
+  version = "4.9.2";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner=pname;
+    repo=pname;
+    rev="v"+version;
```

Change to
```suggestion
    rev="refs/tags/v${version}";
```
or at least to
```suggestion
    rev="v${version}";
```